### PR TITLE
Sanitize the color field to allow using hex codes

### LINKED ISSUE #153: Allow hash in color codes
It would be awesome to allow configuring colors in `#112233` format, because it plays nicer with [IDE plugins](https://github.com/enyancc/vscode-ext-color-highlight) and makes previewing colors easier.
Screen Shot 2022-01-31 at 4 44 58 PM

### PR DESCRIPTION
**Pull request by pjpires**
Found myself struggling with the color field, as I was unfamiliar with the underlying GitHub API.

This PR fixes #153 and adds a test scenario for hex code colors.
Replaces and closes #172.

### CODE REVIEW

File: src/labeler.ts [hunk 1]
```diff
@@ -334,6 +337,10 @@ export class Labeler {
     core.info(`👉 Current labels\n${yaml.dump(labels).toString()}`);
   }
 
+  private sanitizeColorString(color: string) {
+    return color.replace('#', '');
```
Comment (crazy-max) [inline src/labeler.ts:341]: Can we use a regex instead: `color.replace(/^#/, '');`

Comment (pjpires) [reply src/labeler.ts]: That sounds good. 👌 Just pushed the changes!

**Review by crazy-max — CHANGES_REQUESTED** [review]
Looks good but can you:
* squash your commits
* run `docker buildx bake pre-checkin` per https://github.com/crazy-max/ghaction-github-labeler/blob/master/.github/CONTRIBUTING.md#submitting-a-pull-request
* push commit with generated files changes similar to https://github.com/crazy-max/ghaction-github-labeler/pull/187/commits with commit message `update generated content`

Thanks!

**Review by crazy-max — APPROVED** [review]
LGTM thanks!

**Pull request merged**